### ISSUE TITLE
Images in docs aren't accessible (need better alt text)

### ISSUE THREAD

**Issue by DeeDeeG** [OpenVisualCloud/SVT-AV1#1029]
Hi,

I noticed some images in the docs are labeled simply "math", whereas if it is a PNG of the symbol "α" we can change the alt text to be "alpha" or "α" to make that image accessible to screen reader users. (Or in some cases, replace the image with text, e.g. the Unicode α, which is even more accessible, as well as copy-pasteable, IMO.)

(I can volunteer to contribute toward doing this as and when I get the time.)

**Comment by tianjunwork** [comment]
Thank your for volunteering:)
The images in the docs look fine to me. Could you tell me which one is not accessible?

**Comment by DeeDeeG** [comment]
Hi @tianjunwork,

I was thinking of blind computer users. They often use a screen reader to read aloud what's on the screen. But images generally can't be spoken aloud, unless they have text associated with them.

https://webaim.org/techniques/alttext/

So, on GitHub using MarkDown, this would be an image tag with generic alt text ("math"):

`![math](http://latex.codecogs.com/gif.latex?\alpha)` ------> ![math](http://latex.codecogs.com/gif.latex?\alpha)

I am proposing to add more-specific alt text for screen reader users:

`![alpha](http://latex.codecogs.com/gif.latex?\alpha)` ------> ![alpha](http://latex.codecogs.com/gif.latex?\alpha)

---

(You can right click either image above, and "Inspect" to read the alt text.)

Something like this:

```html
<img src="https://camo.githubusercontent.com/a783fa4a3e1f0674c9a0a81de6a3d25fef212b5d/687474703a2f2f6c617465782e636f6465636f67732e636f6d2f6769662e6c617465783f253543616c706861" alt="alpha" data-canonical-src="http://latex.codecogs.com/gif.latex?%5Calpha" style="max-width:100%;">
```

Specifically this bit:
```html
alt="alpha"
```

---

With this added, if there happen to be blind/visually impaired computer users visiting this documentation, using screen-reader software, it will speak the images out loud. The images seem to mostly be formulas and symbols that explain algorithms? This seems like important info that screen reader users wouldn't want to miss out on.

I'm honestly not sure if anyone has tried to read these docs with a screen reader, but I thought correcting the accessible alt text wouldn't hurt.

**Comment by tianjunwork** [comment]
Hi @DeeDeeG , I see, thank you for explaining how it works. It is very thoughtful! Thank you for the suggestion.
I wonder if you would like to contribute to make the change:)

**Comment by DeeDeeG** [comment]
I will have to see when I have some time available, but I would be happy to try then.

**Comment by hassount** [comment]
closing this, please reopen if the issue persists

**Comment by DeeDeeG** [comment]
For the record, I think the task of properly adding alt-text was too challenging for me, every time I looked back into this.

For someone with knowledge of how these formulas could be said out loud, providing alt text would be appreciated by any screen reader users. For an example, the proper alt text for a fraction 1/2 could be something like "one over two" or "one divided by two" or "fraction with the numerator one and the denominator two". Whatever makes it clear, ideally with a mathematically unambiguous phrasing.

Some of the math in the docs are so beyond my level of math knowledge I wouldn't know how to begin transcribing them to speech-like wording like that.
